packages/asset-buyer/src/utils/order_provider_response_processor.ts

```diff
@@ -101,7 +101,7 @@ function filterOutExpiredAndNonOpenOrders(
  */
 function getValidOrdersWithRemainingFillableMakerAssetAmountsFromOnChain(
     inputOrders: SignedOrder[],
-    ordersAndTradersInfo: OrderAndTraderInfo[],
+    ordersAndTradersInfo: any[],
```

Comment:
I think ideally we would keep this as a specific type, but given `asset-swapper` is basically replacing this package seems fine